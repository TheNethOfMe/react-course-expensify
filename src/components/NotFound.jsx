import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <div>
    404 Sorry, we can't find that. - 
    <Link to="/">Go Home</Link>
  </div>
);

export default NotFound;