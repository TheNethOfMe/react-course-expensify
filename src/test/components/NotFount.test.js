import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound.jsx';

test('should render the not found page', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});