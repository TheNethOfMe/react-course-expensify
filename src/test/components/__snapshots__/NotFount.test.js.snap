// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the not found page 1`] = `
<div>
  404 Sorry, we can't find that. -
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
